src/lib/auth/authconf.rb

```diff
@@ -475,12 +475,18 @@ def ldap_export
 
         # Set configuration for LDAP from exported objects.
         def ldap_import(exported_conf)
-            @ldap_conf = exported_conf['conf']
-            @ldap_conf = {} if @ldap_conf.nil?
-            @ldap_pam = exported_conf['pam']
-            @ldap_pam = false if @ldap_pam.nil?
-            @ldap_nss = exported_conf['nss']
-            @ldap_nss = [] if @ldap_nss.nil?
+            if exported_conf.nil?
```

Comment:
np: I would use fetch for this initialization, something like:

```ruby
@ldap_conf = (exported_conf || {}).fetch('conf, {})
@ldap_pam = @ldap_conf.fetch('pam', false)
@ldap_nss = @ldap_conf.fetch('nss', [])
```

Reply:
I overlooked @dgdavid suggestions (https://github.com/yast/yast-auth-client/pull/102#discussion_r626572201)

Reply:
Thanks @teclator and @dgdavid for the suggestions, they are highly appreciated. I am a completely ruby newbie :-)